package.json

```diff
@@ -25,8 +25,6 @@
     "build:dev": "node development/build/index.js",
     "start:test": "yarn env:e2e build:dev testDev --apply-lavamoat=false",
     "start:test:flask": "yarn start:test --build-type flask --apply-lavamoat=false",
-    "start:test:mv2:flask": "ENABLE_MV3=false yarn start:test:flask --apply-lavamoat=false --snow=false",
```

Comment:
Why were these removed?

Reply:
I could be wrong, but I think these commands are no longer necessary.  I should explicitly test though.

Reply:
Whoops, you were correct... I put them back in